### PR TITLE
EdgeDB - Partner and Organization Queries

### PR DESCRIPTION
**Pull request by brentkulwicki**
## Reason for this PR
Migrate Partner and Organization Queries to edgedb

## Ready for review checklist
- [x] Change the task url above to the actual Monday task
- [N/A] Add/update tests if needed
- [x] Add reviewers to this PR

### CODE REVIEW

**Review by CarsonF — COMMENTED** [review]
Want to rebase this and continue here @brentkulwicki?
The yarn changes should be reverted.

There should be other examples of edgedb repos you can follow now.

**Comment by brentkulwicki** [comment]
> Want to rebase this and continue here @brentkulwicki? The yarn changes should be reverted.
> 
> There should be other examples of edgedb repos you can follow now.

Any notes on here? Maybe this will be fine now that I am on a new machine, but I felt like this one was done (outside of all the errors I had that I _thought_ were the issues locally)

File: src/components/organization/organization.repository.ts [hunk 1]
```diff
@@ -46,11 +45,9 @@ export class OrganizationRepository extends DtoRepository<
     return await query.first();
   }
 
-  async update(
-    existing: Organization,
-    changes: ChangesOf<Organization, UpdateOrganization>,
-  ) {
-    return await this.updateProperties(existing, changes);
+  async update(changes: UpdateOrganization, session: Session) {
+    await this.updateProperties({ id: changes.id }, changes);
```
Comment (brentkulwicki) [inline src/components/organization/organization.repository.ts:49]: @CarsonF - Struggling to figure out the types here. It does not like `changes` on this line: 
TS2345: Argument of type `UpdateOrganization` is not assignable to parameter of type `DbChanges<Organization>`
Type `UpdateOrganization` is not assignable to type
```
Partial<Record<'id' | '__typename' | 'createdAt' | 'canDelete' | 'scope', never>>
```
Types of property `id` are incompatible.
Type `ID` is not assignable to type `undefined`

Comment (CarsonF) [reply src/components/organization/organization.repository.ts:49]: It's say you are passing `id` in as a change and you should not be. You can spread it out of the object.
```ts
  async update(input: UpdateOrganization, session: Session) {
    const { id, ...changes } = input;
    await this.updateProperties({ id }, changes);
    return await this.readOne(id, session);
  }
```

File: src/components/partner/partner.service.ts [hunk 2]
```diff
@@ -134,6 +139,7 @@ export class PartnerService {
     }
 
     const changes = this.repo.getActualChanges(partner, input);
+    // comment of where my issue is - yea, I got issues... so what? 😂
```
Comment (brentkulwicki) [inline src/components/partner/partner.service.ts:142]: 👇 This is where I am talking about in slack @CarsonF @bryanjnelson

**Comment by brentkulwicki** [comment]
Still some fixes to make here 🤔

File: src/components/partner/partner.resolver.ts [hunk 3]
```diff
@@ -78,7 +78,7 @@ export class PartnerResolver {
     @Parent() partner: Partner,
     @Loader(OrganizationLoader) organizations: LoaderOf<OrganizationLoader>,
   ): Promise<SecuredOrganization> {
-    return await mapSecuredValue(partner.organization, (id) =>
+    return await mapSecuredValue(partner.organization, ({ id }) =>
```
Comment (brentkulwicki) [inline src/components/partner/partner.resolver.ts:81]: Hey @CarsonF - Any advice here? So I need to go deeper and update this? I don't feel like I should have to update this at all, right? So maybe I made a bad change elsewhere. This only fails for neo4j.

Comment (CarsonF) [reply src/components/partner/partner.resolver.ts:81]: You changed the DTO but didn't change the cypher queries to reflect the shape change.
https://github.com/SeedCompany/cord-api-v3/blob/c70dcf7e9cf1493b015b4f9c9932a3729579df4e/src/components/partnership/partnership.repository.ts#L160-L162
https://github.com/SeedCompany/cord-api-v3/commit/2a04d882aaa99da9d91958c42cd83111a2b594ba#diff-66096aa79c78355d9c374f6ff9e642df1084d4922af8e42fc2a684ea67eff990

Comment (brentkulwicki) [reply src/components/partner/partner.resolver.ts:81]: Didn't even think about the whole "partnership" aspect of it 🤦

Comment (brentkulwicki) [reply src/components/partner/partner.resolver.ts:81]: Wait, did you mean partnership or partner? @CarsonF

Comment (CarsonF) [reply src/components/partner/partner.resolver.ts:81]: Sorry the relevant neo4j repos. partner & org

**Review by CarsonF — APPROVED** [review]

**Pull request merged**